doc/docker.md

```diff
@@ -14,6 +14,18 @@ docker run  --net=host -e LOG_LEVEL=INFO can-provider:latest --server-type kuksa
 docker run  --net=host -e LOG_LEVEL=INFO can-provider:latest --server-type kuksa_val_server
 ```
 
+## Building for a different architecture
+
+You can build for a different architecture by using the `--platform` argument.
+You might need to install `qemu-user-static` depending on your distribution.
+
+```console
+docker build -f Dockerfile --progress=plain --build-arg --platform=linux/arm64 -t can-provider:latest .
```

Comment:
I am not so sure if, this does "all the necessary things", I would prefer we just give the buildx version

Reply:
As I understand from https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/ and https://github.com/docker/cli/pull/3314 the two commands are today equal.

So then it becomes more a question of which syntax we want to promote :-)

I tested building with both, I however only verified that it reported the right architecture.

Reply:
If it is equivalent, what is `--build-arg` doing?  In any case "buildx" might then be more compatible, and also working on docker(-cli?)  < 23.00.

Reply:
That is a type, but I can remove the first alternative